package.json

```diff
@@ -26,6 +26,9 @@
     "lint:js": "eslint --cache .",
     "lint:spelling": "cspell \"**/*.*\"",
     "lint": "npm-run-all -l -p \"lint:**\"",
+    "fix:js": "npm run lint:js -- --fix",
+    "fix:prettier": "npm run lint:prettier -- --write",
+    "fix": "npm-run-all -l -p \"fix:**\"",
```

Comment:
hm, I think I was wrong, we can't use `-p` flag with prettier and eslint, we should run eslint firstly and then prettier

Reply:
```suggestion
    "fix": "npm-run-all -l fix:js fix:prettier",
```